src/babylon.scene.ts

```diff
@@ -1426,10 +1424,13 @@
             if (camera) {
                 return camera;
             }
-
-            var bone = this.getBoneByName(name);
-
-            return bone;
+            var bone = null;
+            for (var skeletonIndex = 0, cache = this.skeletons.length; skeletonIndex < cache; skeletonIndex++) {
+                var skeleton = this.skeletons[skeletonIndex];
+		bone = this.getBoneByName(name, skeleton);

```

Comment:
If a bone was found return it, otherwise it will traverse the entire skeleton index for no reason (performance reduction)